napari/_qt/qt_resources/_icons.py

```diff
@@ -0,0 +1,317 @@
+import os
```

Comment:
It might be helpful to indicate a little more clearly in this file which methods are intended to ultimately be user facing vs which are more internal methods used by those other ones, for example `temporary_qrc_file` I don't think anyone else would ever really need to call themselves? It doesn't need to be private because the whole file is private, it could be an additional comment in the doc string or maybe a reorganisation of this file to have the public functions first and then the helpers or visa-versa, but it took a couple of reads and searching in the diff to get a handle on what gets used in other places vs what gets used internally